### ISSUE TITLE
I just setup the first backup mission and show the error "The passphrase cannot be changed for an existing backup" when i clicked "run now"

### ISSUE THREAD

**Issue by katfionn** [duplicati/duplicati#5095]
- [x] I have searched open and closed issues for duplicates.
- [x] I have searched the [forum](https://forum.duplicati.com) for related topics.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.7.1
- **Operating system**: Debian 11
- **Backend**: WebDAV

## Description

I just setup the first backup mission and show the error "The passphrase cannot be changed for an existing backup" when i clicked "run now". The webdev floder is also empty.

## Steps to reproduce

1. Replace me
2. Replace me
3. ...

- **Actual result**:
i can't backup my data.
- **Expected result**:
when i clicked the button "run now", it start to backup.

## Screenshots

![image](https://github.com/duplicati/duplicati/assets/136874302/7a315c4b-5869-4fb2-86f4-3a1da94a9b75)

## Debug log

i can't read the log, just don't know how, here's what i got, if u need more, pls let me know

`duplicati  |   at Duplicati.Server.WebServer.RESTMethods.RemoteOperation.TestConnection (System.String url, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x000b7] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.RemoteOperation.POST (System.String key, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x00091] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTHandler.DoProcess (Duplicati.Server.WebServer.RESTMethods.RequestInfo info, System.String method, System.String module, System.String key) [0x00289] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  | Mono.Data.Sqlite.SqliteException (0x80004005): SQLite error
duplicati  | no such table: LogData
duplicati  |   at Mono.Data.Sqlite.SQLite3.Prepare (Mono.Data.Sqlite.SqliteConnection cnn, System.String strSql, Mono.Data.Sqlite.SqliteStatement previous, System.UInt32 timeoutMS, System.String& strRemain) [0x001f8] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.BuildNextCommand () [0x000d3] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.GetStatement (System.Int32 index) [0x00008] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at (wrapper remoting-invoke-with-check) Mono.Data.Sqlite.SqliteCommand.GetStatement(int)
duplicati  |   at Mono.Data.Sqlite.SqliteDataReader.NextResult () [0x000b3] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteDataReader..ctor (Mono.Data.Sqlite.SqliteCommand cmd, System.Data.CommandBehavior behave) [0x0004e] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at (wrapper remoting-invoke-with-check) Mono.Data.Sqlite.SqliteDataReader..ctor(Mono.Data.Sqlite.SqliteCommand,System.Data.CommandBehavior)
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.ExecuteReader (System.Data.CommandBehavior behavior) [0x00006] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.ExecuteDbDataReader (System.Data.CommandBehavior behavior) [0x00000] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader () [0x00000] in <5cacff0b76ce45dfa780aae0bc1e2fd5>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.LogData.DumpTable (System.Data.IDbCommand cmd, System.String tablename, System.String pagingfield, System.String offset_str, System.String pagesize_str) [0x000da] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.Backup.FetchLogData (Duplicati.Server.Serialization.Interface.IBackup backup, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x00053] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.Backup.GET (System.String key, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x001e4] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTHandler.DoProcess (Duplicati.Server.WebServer.RESTMethods.RequestInfo info, System.String method, System.String module, System.String key) [0x00146] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  | Mono.Data.Sqlite.SqliteException (0x80004005): SQLite error
duplicati  | no such table: RemoteOperation
duplicati  |   at Mono.Data.Sqlite.SQLite3.Prepare (Mono.Data.Sqlite.SqliteConnection cnn, System.String strSql, Mono.Data.Sqlite.SqliteStatement previous, System.UInt32 timeoutMS, System.String& strRemain) [0x001f8] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.BuildNextCommand () [0x000d3] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.GetStatement (System.Int32 index) [0x00008] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at (wrapper remoting-invoke-with-check) Mono.Data.Sqlite.SqliteCommand.GetStatement(int)
duplicati  |   at Mono.Data.Sqlite.SqliteDataReader.NextResult () [0x000b3] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteDataReader..ctor (Mono.Data.Sqlite.SqliteCommand cmd, System.Data.CommandBehavior behave) [0x0004e] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at (wrapper remoting-invoke-with-check) Mono.Data.Sqlite.SqliteDataReader..ctor(Mono.Data.Sqlite.SqliteCommand,System.Data.CommandBehavior)
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.ExecuteReader (System.Data.CommandBehavior behavior) [0x00006] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at Mono.Data.Sqlite.SqliteCommand.ExecuteDbDataReader (System.Data.CommandBehavior behavior) [0x00000] in <0e0c808ba41a4844814b82eb7ba357a3>:0 
duplicati  |   at System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader () [0x00000] in <5cacff0b76ce45dfa780aae0bc1e2fd5>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.LogData.DumpTable (System.Data.IDbCommand cmd, System.String tablename, System.String pagingfield, System.String offset_str, System.String pagesize_str) [0x000da] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.Backup.FetchRemoteLogData (Duplicati.Server.Serialization.Interface.IBackup backup, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x00052] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTMethods.Backup.GET (System.String key, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x001ed] in <30a34d71126b48248d040dda634ddad9>:0 
duplicati  |   at Duplicati.Server.WebServer.RESTHandler.DoProcess (Duplicati.Server.WebServer.RESTMethods.RequestInfo info, System.String method, System.String module, System.String key) [0x00146] in <30a34d71126b48248d040dda634ddad9>:0 `

**Comment by katfionn** [comment]
it just won't work, i run it based on docker, and i want to backup the whole system. i already remove it from docker.

**Comment by ts678** [comment]
> "The passphrase cannot be changed for an existing backup" when i clicked "run now"

is as documented if you set advanced option [allow-passphrase-change](https://duplicati.readthedocs.io/en/latest/06-advanced-options/#allow-passphrase) and run backup.

What, if anything, are you doing with passphrases and encryption on General screen 1?

> i already remove it from docker.

Whose image is it? The one from the Duplicati team runs as root, so gets simpler access.

[https://hub.docker.com/r/duplicati/duplicati](https://hub.docker.com/r/duplicati/duplicati)
[https://hub.docker.com/r/linuxserver/duplicati](https://hub.docker.com/r/linuxserver/duplicati)

Both give directions on what host folders must be provided. Make sure access is allowed.
About --> System info shows `UserName` which would be a clue. Or use another method.

```
SQLite error duplicati  | no such table: LogData
SQLite error duplicati  | no such table: RemoteOperation
```

are not normal. Possibly some buttons were pushed, or access was not provided properly.
[Database management](https://duplicati.readthedocs.io/en/latest/03-using-the-graphical-user-interface/#database-management) will show where database appears inside Docker. Is it in host too?
There will probably be a Duplicati-server.sqlite database nearby, for job settings and stats.

> I just setup the first backup
> The webdev floder is also empty.

After you look around to see if you have any databases on the host (if so, what sizes) and
if you're sure nothing ever got backed up, maybe use the Database page Delete, and Run.

**Comment by katfionn** [comment]
> > "The passphrase cannot be changed for an existing backup" when i clicked "run now"
> 
> is as documented if you set advanced option [allow-passphrase-change](https://duplicati.readthedocs.io/en/latest/06-advanced-options/#allow-passphrase) and run backup.
> 
> What, if anything, are you doing with passphrases and encryption on General screen 1?
> 
> > i already remove it from docker.
> 
> Whose image is it? The one from the Duplicati team runs as root, so gets simpler access.
> 
> https://hub.docker.com/r/duplicati/duplicati https://hub.docker.com/r/linuxserver/duplicati
> 
> Both give directions on what host folders must be provided. Make sure access is allowed. About --> System info shows `UserName` which would be a clue. Or use another method.
> 
> ```
> SQLite error duplicati  | no such table: LogData
> SQLite error duplicati  | no such table: RemoteOperation
> ```
> 
> are not normal. Possibly some buttons were pushed, or access was not provided properly. [Database management](https://duplicati.readthedocs.io/en/latest/03-using-the-graphical-user-interface/#database-management) will show where database appears inside Docker. Is it in host too? There will probably be a Duplicati-server.sqlite database nearby, for job settings and stats.
> 
> > I just setup the first backup
> > The webdev floder is also empty.
> 
> After you look around to see if you have any databases on the host (if so, what sizes) and if you're sure nothing ever got backed up, maybe use the Database page Delete, and Run.

---

OMG, i'm a idiot, thank you to remind me get the official docker Image, i was get it from Casaos. Thanks again.

**Comment by justauserid** [comment]
> is as documented if you set advanced option [allow-passphrase-change](https://duplicati.readthedocs.io/en/latest/06-advanced-options/#allow-passphrase) and run backup.

This is weird though. I was literally trying the app, first time ever. Was selecting back destination, source files, retention period and was given this option and I selected it and got the same error. No log, nothing else, just this message `The passphrase cannot be changed for an existing backup`.

Now, my understanding was - it will be applied when I try to change the backup. I don't understand why this would be shown as an advanced option to be set and then picked it a command where it should not be picked i.e as a arg/param. I mean if this doesn't work in backup then why is it shown to be selected, or why does it get included in the command? 

Again, I had ZERO backups created. In fact finished setup and while the first scheduled run was some 30-40 min away I tried to give it a try but doing my first back as - manually backuo right then.
